lib/contestant.rb

```diff
@@ -0,0 +1,32 @@
+require "./lib/game"
+require 'pry'
+
+class Contestant
+attr_reader :first_name, :last_name, :age, :state_of_residence, :spending_money, :game_interests_list
+
+  def initialize(contestant_data)
+    @first_name = contestant_data[:first_name]
+    @last_name = contestant_data[:last_name]
+    @age = contestant_data[:age]
+    @state_of_residence = contestant_data[:state_of_residence]
+    @spending_money = contestant_data[:spending_money]
+    @game_interests_list = []
+  end
+
+  def full_name
+    [@first_name, @last_name].join(' ')
+  end
+
+  def out_of_state?
+    !@state_of_residence == "CO"
```

Comment:
Not quite, for this to run, we need the condition to run within parentheses 
`!(@state_of_residence == "CO")` so that the not operator flips the return of the condition instead of the what is held in the instance variable. Another way that reads a bit better is 
`@state_of_residence != "CO"`